### ISSUE TITLE
Update box2 to 4.0

### ISSUE THREAD

**Issue by arj03** [ssbc/ssb-db2#414]
Makes it easier to work with tribes2.

**Comment by staltz** [comment]
@arj03 Since box2 is a dependency in ssb-db2, and box2 `4.0` nowadays doesn't work unless you have ssb-meta-feeds, this means we have to make ssb-meta-feeds a dependency in ssb-db2, right?

It's a bit confusing, because ssb-db2 is a dependency in ssb-meta-feeds.

It would be easier if `ssb-db2/index.js` would be just a "distro" of ssb-db2, perhaps in a separate module `ssb-db2-starter-kit` (I'm not serious about this idea), then we would solve the dependency chicken-and-egg. As you said, the chicken-and-egg issue makes it hard to use some ssb-db2 in some projects. In fact, we also have the ssb-fixtures chicken-and-egg with db2.

**Comment by arj03** [comment]
Yeah, I'm maybe more included to moving some of these things out of db2. Instead of the distro thing we could maybe just document it really well in the readme. Like, if you just want classic then include classic + box1. If you want "proper" box2 then include tribes, metafeeds and box etc.

**Comment by staltz** [comment]
Yeah, I sympathize with that plan.

And just a note, it would be a breaking change.
